Index: swapspace-1.10/Makefile
===================================================================
--- swapspace-1.10.orig/Makefile	2008-09-17 22:23:24.980226538 +0300
+++ swapspace-1.10/Makefile	2008-09-17 22:22:29.068926150 +0300
@@ -3,9 +3,9 @@
 SWAPPARENT=/var/lib
 SWAPDIR=$(SWAPPARENT)/swapspace
 
-all: VERSION DATE
-	+$(MAKE) -C src VERSION="`cat VERSION`" DATE="`cat DATE`"
-	+$(MAKE) -C doc VERSION="`cat VERSION`" DATE="`cat DATE`"
+all: debian/VERSION debian/DATE
+	+$(MAKE) -C src VERSION="`cat debian/VERSION`" DATE="`cat debian/DATE`"
+	+$(MAKE) -C doc VERSION="`cat debian/VERSION`" DATE="`cat debian/DATE`"
 
 clean:
 	+$(MAKE) -C src clean
@@ -18,7 +18,7 @@
 	install -d -m700 $(DESTDIR)/$(SWAPDIR)
 	install -d $(DESTDIR)/usr/sbin $(DESTDIR)/etc $(DESTDIR)/etc/init.d
 	install -d $(DESTDIR)/usr/share/man/man8
-	strip src/swapspace || true
+#	strip src/swapspace || true
 	install -m755 src/swapspace $(DESTDIR)/usr/sbin
 	install -m644 swapspace.conf $(DESTDIR)/etc
 	install -m744 debian/swapspace.init $(DESTDIR)/etc/init.d/swapspace
@@ -41,10 +41,10 @@
 
 
 # Derive version number and last change date from Debian changelog
-VERSION: debian/changelog
+debian/VERSION: debian/changelog
 	head -n 1 $< | sed -e 's/^.*(\([0-9][^)]*\)).*/\1/' >$@
 
-DATE: debian/changelog
+debian/DATE: debian/changelog
 	grep '^ -- ' $< | sed -e 's/^.*>  \([MTWFS][a-z]\{2\}, \{1,2\}[[:digit:]]\{1,2\} [JFBAMSOND][a-z]* 2[[:digit:]]\{3\}\) .*/\1/' | head -n 1 >$@
 
 .PHONY: all clean install uninstall
Index: swapspace-1.10/src/Makefile
===================================================================
--- swapspace-1.10.orig/src/Makefile	2008-09-17 22:23:25.028226633 +0300
+++ swapspace-1.10/src/Makefile	2008-09-17 22:23:39.968035952 +0300
@@ -3,7 +3,7 @@
 # Assumes we're using gcc
 # swapspace is written in C99, but with at least one extension that isn't
 # supported in gcc's C99 mode: sigaction() (though it is part of POSIX).
-CFLAGS += --std=gnu99 -Wall --pedantic -Wshadow -O2 -g
+CFLAGS += --std=gnu99 -Wall --pedantic -Wshadow -O2 -g $(DEB_FLAGS)
 CPPFLAGS += -DSUPPORT_LARGE_FILES -DVERSION="\"$(VERSION)\"" -DDATE="\"$(DATE)\""
 RM=rm -f
 
